tests/handlers/records-write.spec.ts

```diff
@@ -34,1487 +34,1490 @@ import { RecordsWrite } from '../../src/interfaces/records-write.js';
 import { RecordsWriteHandler } from '../../src/handlers/records-write.js';
 import { stubInterface } from 'ts-sinon';
 import { TestDataGenerator } from '../utils/test-data-generator.js';
-import { TestStoreInitializer } from '../test-store-initializer.js';
+import { TestStores } from '../test-stores.js';
 import { TestStubGenerator } from '../utils/test-stub-generator.js';
 
 import { Encryption, EncryptionAlgorithm } from '../../src/utils/encryption.js';
 
 chai.use(chaiAsPromised);
 
-describe('RecordsWriteHandler.handle()', () => {
-  let didResolver: DidResolver;
-  let messageStore: MessageStore;
-  let dataStore: DataStore;
-  let eventLog: EventLog;
-  let dwn: Dwn;
+export function testRecordsWriteHandler(): void {
+  describe('RecordsWriteHandler.handle()', async () => {
+    let didResolver: DidResolver;
+    let messageStore: MessageStore;
+    let dataStore: DataStore;
+    let eventLog: EventLog;
+    let dwn: Dwn;
 
-  describe('functional tests', () => {
+    describe.only('functional tests', () => {
```

Comment:
```suggestion
    describe('functional tests', () => {
```

Reply:
Thanks for catching this! Didn't realize `.only` stopped working as I was using it to test my changes quickly. Fixed.